content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -408,3 +408,84 @@ second ensure that [keys and certificates are being delivered](#keys-and-certifi
 
 If everything appears to be working so far, the next step is to verify that the right [authentication policy](/docs/tasks/security/authentication/authn-policy/)
 is applied and the right destination rules are in place.
+
+### Auto mTLS
+
+If you suspect [auto mTLS](/docs/reference/glossary#auto-mtls) is not working
+as expected, please first read the
```

Comment:
Should reword this, comes across as a little RTFM. Why are you asking them to read the documentation, is there something in particular in there to help them troubleshoot?

Reply:
+1, how would user know when it is not working?  I guess service A won't be able to call service B with 503?

Reply:
updated.